app/lib/three_scale/analytics/user_tracking.rb

```diff
@@ -231,7 +239,7 @@ def identified?
 
       def segment_client(options = {})
         merged_options = segment_options.deep_merge(options)
-        Segment.with_options(merged_options) { |segment| return segment }
+        Segment.with_options(merged_options) { |segment| segment }
```

Comment:
Hmmm wait, `return` will make the calling method return so careful with that!

Reply:
This definitely requires a test 😝